torch/_export/logical_schema.py

```diff
@@ -269,6 +269,18 @@ class Graph:
     symint_values: List[SymIntValue]
 
 
+@dataclass(init=False)
+class Epilogue:
```

Comment:
an alternative design I think also works here:
```Python
class Epilogue:
    copies: List[...]
    index_puts: List[...]
```
and inputs data can be expressed implicitly inside `...` part.